### DISCUSSION TITLE
Difference between bundle relation update and upgrade.

### DISCUSSION THREAD

**Discussion by ds-andreasor** [wixtoolset/issues#8064] [Questions] [answered]
While I was checkin some of the detect events I found that a related bundle can be of type "Upgrade" or "Update".

Is it possible to create a bundle update?
This would mean that the ProviderKey and the UpgradeCode must stay the same while the Version must be higher.
If this is used the packages in the chain must stay the same as well?
It would not be possible to add or remove packages from the chain.

The upgrade bundle would change the ProviderKey and only the UpgradeCode stays the same while the Version must be higher.
If no explicit ProviderKey is authored one will be generated with the bundle build?
If this is used it is possible to add/remove packages from the chain in the related bundle.

**Comment by barnson** [comment, marked as answer]
Updates are actions that tell Burn to download an upgrade bundle. Bundles don't have the small update, minor upgrade, and major upgrade distinctions that MSI does.

**Reply by ds-andreasor** [reply]
That was what I was seeing in the logs of my tests for this.
Maybe the documentation can be made clearer for this, so that intellisense already explains this.
